changelog/unreleased/enhancement-update-ods

```diff
@@ -1,37 +1,37 @@
-- Change - Remove OcAppSideBar component: https://github.com/owncloud/owncloud-design-system/pull/1810
-- Change - Remove OcAppBar component: https://github.com/owncloud/owncloud-design-system/pull/1810
-- Change - Remove implicit ODS registration: https://github.com/owncloud/owncloud-design-system/pull/1848
-- Change - Remove oc-table-files from ods: https://github.com/owncloud/owncloud-design-system/pull/1817
-- Change - Remove OcGrid options: https://github.com/owncloud/owncloud-design-system/pull/1658
-- Change - Move OcSidebarNav and OcSidebarNavItem to web: https://github.com/owncloud/web/issues/6036
-- Change - Remove UiKit: https://github.com/owncloud/owncloud-design-system/pull/1658
-- Change - Remove unused props for unstyled components: https://github.com/owncloud/owncloud-design-system/pull/1795
-- Change - Use remixicons for redesign: https://github.com/owncloud/owncloud-design-system/pull/1826
-- Enhancement - Make Vue-Composition-API available: https://github.com/owncloud/owncloud-design-system/pull/1848
-- Enhancement - Export mappings of types, icons and colors of resources: https://github.com/owncloud/owncloud-design-system/pull/1920
-- Enhancement - Fix OcAvatar line-height: https://github.com/owncloud/owncloud-design-system/pull/1810
-- Enhancement - Add option to render table cells lazy: https://github.com/owncloud/owncloud-design-system/pull/1848
-- Enhancement - Make OcDrop rounded: https://github.com/owncloud/owncloud-design-system/pull/1881
-- Enhancement - Change background color of OcDrop: https://github.com/owncloud/owncloud-design-system/pull/1919
-- Enhancement - Improve OcList: https://github.com/owncloud/owncloud-design-system/pull/1881
-- Enhancement - Show path / parent folder to distinguish files: https://github.com/owncloud/web/issues/5953
-- Enhancement - Redesign Filetype icons: https://github.com/owncloud/web/issues/6278
-- Enhancement - Adjust OcSearchBar to new design: https://github.com/owncloud/owncloud-design-system/pull/1810/
-- Enhancement - Sizes: https://github.com/owncloud/owncloud-design-system/pull/1858
-- Enhancement - Add svg icon for spaces: https://github.com/owncloud/owncloud-design-system/pull/1846
-- Enhancement - OcTable header alignment: https://github.com/owncloud/owncloud-design-system/pull/1922
-- Enhancement - Use Roboto font: https://github.com/owncloud/owncloud-design-system/pull/1876
-- Bugfix - Missing OcDrop shadow: https://github.com/owncloud/owncloud-design-system/pull/1926
-- Bugfix - OcNotification positioning: https://github.com/owncloud/owncloud-design-system/pull/1658
-- Bugfix - Rename GhostElement: https://github.com/owncloud/owncloud-design-system/pull/1845
-- Bugfix - OcTooltip isn't reactive: https://github.com/owncloud/owncloud-design-system/pull/1863
+* Change - Drop Internet Explorer support: [#1909](https://github.com/owncloud/owncloud-design-system/pull/1909)
```

Comment:
If you change back to dashes it gives a better impression of what changes in this PR specifically;)

Reply:
incl. removing the PR/issue `[#nmbr]` and the braces

Reply:
ah, dang. just copied it from the ODS changelog. fixing, gimme a minute...

Reply:
I was quicker haha